### PR TITLE
Error message updates

### PR DESCRIPTION
**Pull request by amichaelparker**
### Summary
Updated error messages with suggested text from [issue #1210 ](https://github.com/cerner/terra-core/issues/1210).

### CODE REVIEW

File: packages/terra-table/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,8 @@ Changelog
 
 Unreleased
 ----------
+### Changed
+* Changed background color to transparent
```
Comment (ryanthemanuel) [inline packages/terra-table/CHANGELOG.md]: Should this be a separate PR?

File: packages/terra-action-header/src/ActionHeader.jsx [hunk 2]
```diff
@@ -69,7 +69,7 @@ const contextTypes = {
   /* eslint-disable consistent-return */
   intl: (context) => {
     if (context.intl === undefined) {
-      return new Error('Please add locale prop to Base component to load translations');
+      return new Error('Component requires Localization, and must be wrapped in terra-base');
```
Comment (emilyrohrbough) [inline packages/terra-action-header/src/ActionHeader.jsx]: Would it make more sense to state that the component is internationalized and must be wrapped in base rather than saying it requires localization?

File: packages/terra-table/CHANGELOG.md (hunk 1 above)
Comment (yuderekyu) [reply packages/terra-table/CHANGELOG.md]: it's here https://github.com/cerner/terra-core/pull/1758. looks like you branched off that branch instead of master

Comment (amichaelparker) [reply packages/terra-table/CHANGELOG.md]: Worked with @bjankord on reverting commit, should be correct now.

File: packages/terra-action-header/src/ActionHeader.jsx (hunk 2 above)
Comment (amichaelparker) [reply packages/terra-action-header/src/ActionHeader.jsx]: Spoke with @JakeLaCombe regarding error text changes, he agrees with the changes suggested.

File: packages/terra-table/CHANGELOG.md [hunk 3]
```diff
@@ -3,8 +3,6 @@ Changelog
 
 Unreleased
 ----------
-### Changed
```
Comment (ryanthemanuel) [inline packages/terra-table/CHANGELOG.md]: Did this accidentally get removed after a merge?

Comment (amichaelparker) [reply packages/terra-table/CHANGELOG.md]: Yep! Worked with @bjankord on appropriate correction.
